### PR TITLE
Show no projects if there is no projects 

### PR DESCRIPTION
**Pull request by yash15601**
Fixes 
https://github.com/CircuitVerse/CircuitVerse/issues/711

#### Describe the changes you have made in this pr 
Show no projects if there is  no projects 

### Screenshots of the changes (If any) -
Screenshot (223)

### CODE REVIEW

**Comment by Ananthavijay** [comment]
@sal2701 Is this fine? or should there be a motivational message?

**Comment by Shivansh2407** [comment]
Should the target be _blank or should it redirect towards create new project page?

**Comment by yash15601** [comment]
@Shivansh2407 when we click on here it will be redirected to simulator

**Comment by Shivansh2407** [comment]
> @Shivansh2407 when we click on here it will be redirected to simulator

Cool Understood. I had misunderstood it earlier 👍

**Comment by sal2701** [comment]
@Ananthavijay A motivational message or something similar to this [link](https://optinmonster.com/best-404-page-examples/) for Error 404 pages would be great. We can do it at a later stage.
@yash15601 For now we can just move the text in the PR to the center of the page because the page looks very empty and the text is in a corner.

File: app/views/users/logix/index.html.erb [hunk 1]
```diff
@@ -32,6 +32,12 @@
   <div class ="row">
     <h2 class="card-title"><%= @user.name%> - Projects | <a href=  "<%= profile_path(@user)%>"  target="_blank"> View Profile </a> | <a href=  "<%= user_favourites_path(@user)%>"  target="_blank"> View Favourites </a> </h2>
   </div>
+  <% projects = @user.projects %>
+  <% if projects.count == 0 %>
```
Comment (tachyons) [inline app/views/users/logix/index.html.erb]: No need of temporary variable, just @user.projects.blank? Is enough

**Comment by yash15601** [comment]
@tachyons Please review

File: app/views/users/logix/index.html.erb [hunk 2]
```diff
@@ -32,6 +32,11 @@
   <div class ="row">
     <h2 class="card-title"><%= @user.name%> - Projects | <a href=  "<%= profile_path(@user)%>"  target="_blank"> View Profile </a> | <a href=  "<%= user_favourites_path(@user)%>"  target="_blank"> View Favourites </a> </h2>
   </div>
+  <% if @user.projects.blank? %>
+    <div class ="noprojects" style="position: absolute;top: 30%;left: 30%;">
```
Comment (tachyons) [inline app/views/users/logix/index.html.erb]: Use css for styles

File: app/views/users/logix/index.html.erb [hunk 3]
```diff
@@ -32,6 +32,11 @@
...
+      <h5>You dont have any projects. Create one <a href=  "/simulator"  target="_blank"> here </a></h5>
```
Comment (tachyons) [inline app/views/users/logix/index.html.erb]: Use `link_to` helper, Open in same window, not new window

Comment (Ananthavijay) [reply app/views/users/logix/index.html.erb]: @tachyons won't removing ````taget="_blank"```` solve the problem?

Comment (yash15601) [reply app/views/users/logix/index.html.erb]: @tachyons target="_self" will also solve  problem

**Comment by yash15601** [comment]
Done @tachyons  Review

**Comment by yash15601** [comment]
Done @tachyons  Review

File: app/views/users/logix/index.html.erb [hunk 4]
```diff
@@ -100,3 +105,4 @@
         })
     } );
 </script>
+<link rel="stylesheet" type="text/css" href="/css/embed.css">
```
Comment (tachyons) [inline app/views/users/logix/index.html.erb]: Use asset pipeline https://edgeguides.rubyonrails.org/asset_pipeline.html

**Comment by yash15601** [comment]
@tachyons  Done

**Pull request merged**